assembly/src/parsers/mod.rs

```diff
@@ -22,7 +25,21 @@ fn parse_op_token(
     span_ops: &mut Vec<Operation>,
     num_proc_locals: u32,
     decorators: &mut DecoratorList,
+    in_debug_mode: bool,
+    ops_len: &mut usize,
+    dec_len: &mut usize,
```

Comment:
Why do we need to pass these two variables here? It seems like they are overwritten in the first `in_debug_mode` clause below.

Reply:
I thought passing the value by reference would be cheaper than instantiating a variable for ops_len and dec_len each time `parse_op_token` is called in non debug mode since it's the same reference that is being passed every time. Not sure if that is correct or even if it is, if it is worth doing while sacrificing readability.

Reply:
Removed.